internal/exec/engine.go

```diff
@@ -74,6 +80,21 @@ func (e Engine) Run(stageName string) error {
 		e.Logger.Crit("failed to acquire system base config: %v", err)
 		return err
 	}
+	var haveConfig bool
+	// TODO(sohan) Check to see if we already have a
+	// cached config. This is a hack to emit the journald
+	// message only once for the base config. We can remove
+	// this once https://github.com/coreos/ignition/issues/950
+	// will be fixed.
+	if _, err := os.Stat(e.ConfigCache); err != nil && os.IsExist(err) {
```

Comment:
We should log and return an error if the error isn't `IsNotExist`. We should really add a helper utility function for this somewhere...